src/main/java/frc/robot/Config.java

```diff
@@ -467,6 +467,62 @@ public class ArmConfig {
     public static final double MOMENT_TO_VOLTAGE = 0.000005;    
 }
 
+  public class ElevatorConfig {
+    public static final int ELEVATOR_SPARK_CAN_ID = CANID.ELEVATOR;
+    public static final boolean SET_INVERTED = true;
+    public static final boolean INVERT_ENCODER = false;
+
+    public static final int CURRENT_LIMIT = 20;
+
+    public static final double shiftEncoderRange = 10;
+    //offset unit: degrees
+    public static final double elevatorAbsEncoderOffset = Math.toDegrees(3.20433) + 3.0 - shiftEncoderRange;
+
+
+    public static final double MAX_ELEVATOR_EXTENSION = 1000; // Temp value for testing
+    public static final double MIN_ELEVATOR_EXTENSION = -2; // Temp value for testing
+
+
+    //soft limit constant for the elevator
+    public static final float elevator_up_limit = (float) Math.toRadians(MAX_ELEVATOR_EXTENSION + shiftEncoderRange);
+    public static final float elevator_down_limit = (float) Math.toRadians(MIN_ELEVATOR_EXTENSION + shiftEncoderRange);
+    public static final boolean SOFT_LIMIT_ENABLE = true;
+
+    //PID constants
+    public static final double elevator_kP = robotSpecific(2.700000, 0.0, 0.0, 1.4);
```

Comment:
If you want to test it on Beetle, set values to index 2, since Beetle has robot id = 2